shared/actions/fs/common.native.tsx

```diff
@@ -11,11 +11,19 @@ import {saveAttachmentDialog, showShareActionSheetFromURL} from '../platform-spe
 import {types} from '@babel/core'
 
 const pickAndUploadToPromise = (state: TypedState, action: FsGen.PickAndUploadPayload): Promise<any> =>
-  new Promise((resolve, reject) =>
-    ImagePicker.showImagePicker(
+  new Promise((resolve, reject) => {
+    let title = 'Select a Photo'
+    let captureTitle = 'Take Photo...'
```

Comment:
I think design prefer not to capitalize every workd. So "Select a photo" and "Take photo..." here. Same below.